lighthouse/helpers/plates.py

```diff
@@ -369,7 +370,7 @@ def update_mlwh_with_cog_uk_ids(samples: List[Dict[str, str]]) -> None:
                     table.c.result == bindparam("b_result"),
                 )
             )
-            .values(cog_uk_id=bindparam("b_cog_uk_id"))
+            .values(cog_uk_id=bindparam("b_cog_uk_id"), updated_at=datetime.now())
```

Comment:
We have constants for these column headers.  I think we should probably be using them.  So instead of using `**kwargs` you could use `*args` and it would be like this (see https://docs.sqlalchemy.org/en/14/core/dml.html#sqlalchemy.sql.expression.Update.values):

```suggestion
            .values(
                {
                    MLWH_LH_SAMPLE_COG_UK_ID: bindparam("b_cog_uk_id"),
                    MLWH_LH_SAMPLE_UPDATED_AT: datetime.now()
                }
            )
```

Obviously you'll need to import those constants.